hw/ip/aes/data/aes.hjson

```diff
@@ -38,7 +38,7 @@
       desc: '''
         Initial Key Registers.
         Loaded into the internal Full Key register upon starting encryption/decryption of the next block.
-        All key registers must be written at least once when the key is changed, regardless of key length (write 0 for unusued bits).
+        All key registers must be written at least once when the key is changed, regardless of key length (write random data for unused bits).
```

Comment:
I am not entirely aware of what the status of random number/data generator in OT is, and what the interface would be (through CSRs, or it would directly sideload information). Basically if we take DIF AES for example, what would be the best way to get random data, or at the moment we load it with zeros but later on should fix it up to load with random data?

Reply:
My understanding is that there will be a register for the processor to read to get entropy as in the current entropy source (see entropy_src.ES_ENTROPY [here](https://docs.opentitan.org/hw/ip/entropy_src/doc/index.html#register-table), maybe the CSRNG will eventually implement a similar register).

I think for the moment, the DIF should load with zeros or use the register in the entropy source but I don't know if there is software available for this at the moment. Anyway, this will have to be reworked once the real entropy source/CSRNG is available.

Reply:
Thanks!